src/uwtools/drivers/driver.py

```diff
@@ -254,7 +264,7 @@ def _write_runscript(self, path: Path, envvars: Dict[str, str]) -> None:
             envvars=envvars,
             execution=[
                 "time %s" % self._runcmd,
-                "test $? -eq 0 && touch %s/done.%s" % (self._rundir, self._driver_name),
+                "test $? -eq 0 && touch %s.done" % self._runscript_path.name,
```

Comment:
We shouldn't need an absolute path here, as we want this file to be relative to the runscript, i.e. in the run directory, which should already be the case. This chang makes the runscript a little more concise, and also means one less thing to manually change if the run directory is copied to another path.

Also, we may now see a series of closely related files grouped together, e.g. for `upp` submitted to the batch system:
```
runscript.upp
runscript.upp.done
runscript.upp.out
runscript.upp.submit
```
The first is the runscript itself. The second is the empty sentinel file created if the component runs to completion and exits with success status. The third is the combined stdout/stderr streams from the runscript and everything that happens in it. And the last is the output produced by whatever command submits the runscript to the batch scheduler (should contain the batch job ID).